addons/mail/views/mail_template_views.xml

```diff
@@ -13,30 +13,33 @@
                         <button string="Preview"
                                 name="%(mail_template_preview_action)d" type="action"
                                 class="btn-primary" target="new"/>
-                        <button string="Reset Template"
-                                name="%(mail_template_reset_action)d" type="action"
-                                groups="mail.group_mail_template_editor"
-                                invisible="not template_fs"/>
-                        <button string="Add Context Action"
-                                class="btn btn-secondary"
-                                name="create_action" type="object"
-                                groups="base.group_system"
-                                invisible="ref_ir_act_window"
-                                help="Display an option on related documents to open a composition wizard with this template"/>
-                        <button string="Remove Context Action"
-                                class="btn btn-secondary"
-                                name="unlink_action" type="object"
-                                groups="base.group_no_one"
-                                invisible="not ref_ir_act_window"
-                                help="Remove the contextual action to use this template on related documents"/>
+                        <t groups="base.group_no_one">
```

Comment:
Would not put those in debug mode. Last time I saw someone trying to unfuck templates, he was happy to find a "reset template" button and I bet he would not have tried the debug mode to find one.